postgresql/database_test.go

```diff
@@ -57,32 +52,37 @@ var settings = ConnectionURL{
 	Password: password,
 }
 
-var host = flag.String("host", "testserver.local", "Testing server address.")
+var host = flag.String("host", "localhost", "Testing server address.")
```

Comment:
Oh, good idea! I think using environment variables would be a lot more flexible that test flags.